### PR TITLE
Feature/components

### PR DESCRIPTION
**Pull request by Lameck1**
# Description

This PR submit the Calculator component feature

## Tasks completed

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called `components`.
- Inside `components`, create a new `Calculator.js` file.
- In `Calculator.js`, create a React component that matches the given design.
- Don't add any extra design.
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class-based components.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (mricanho) [inline src/App.js]: - [x] This function  should be class-based. The same way you did with the Calculator component. Please fix it.

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @Lameck1, 

Good job so far! Everything works right now, just a quick fix on the App.js and you are good to go🔥🔥

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [x] App.js function should be class-based. The same way you did with the Calculator component. Please fix it.

Check the comments under the review.

## Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

yougotit

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Lameck1** [comment]
Thanks for the heads-up @mricanho.

**Review by henatan99 — APPROVED** [review]
Hi @Lameck1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**